### PR TITLE
Update SIR description.

### PR DESCRIPTION
**Pull request by vext01**

### CODE REVIEW

**Comment by vext01** [comment]
Looks like CI wasn't fully set up on this repo.

File: src/tech/irs.md [hunk 1]
```diff
@@ -8,23 +8,30 @@ already found in rustc:
...
 [externally maintained crate](https://github.com/softdevteam/yk/tree/master/ykpack)
 so that they can be shared by the compiler and the JIT runtime.
 
-SIR lowering
-[is performed here](https://github.com/softdevteam/ykrustc/blob/master/src/librustc_yk_sections/emit_sir.rs).
+The
+[SirCx](https://github.com/softdevteam/ykrustc/blob/master/src/librustc/sir.rs)
+is a per-codegen-unit structure that holds all state related to SIR.
+
+Often we have to lookup SIR constructs (functions, blocks, etc) identified in
```
Comment (ptersilie) [inline src/tech/irs.md]: `etc.`? (with a dot)

File: src/tech/irs.md [hunk 2]
```diff
@@ -8,23 +8,30 @@ already found in rustc:
...
+the codegen only by an opaque LLVM pointer. For this reason the `SirCx` has to
+hold a cache that maps these pointers back to corresponding Sir structures.
+
+### Why is SIR Implemented in the LLVM Codegen?
+
+Initially SIR was generated from the Rust's Middle Intermediate Representation
```
Comment (ptersilie) [inline src/tech/irs.md]: remove the `the` before `Rust's`

**Comment by ptersilie** [comment]
Two typos. Otherwise looks good.

**Comment by vext01** [comment]
Fixed. Squash?

**Comment by ptersilie** [comment]
Please do.

**Comment by vext01** [comment]
squashed

**Pull request merged**